doc/getting-started/getting-started.rst

```diff
@@ -594,9 +597,9 @@ Build ACRN
       .. code-block:: bash
 
          disk="/media/$USER/"$(ls /media/$USER)
-         cp "$disk"/acrn-my_board-shared-2.7.deb ~/acrn-work # TODO update file name
+         cp "$disk"/acrn-my_board-MyConfiguration-3.0-acrn-*.deb ~/acrn-work # TODO update file name
```

Comment:
If that's indeed the right file name now, remove the #TODO comment

Reply:
update